packages/widget-chat/src/modules/redux-spark/spark.js

```diff
@@ -3,6 +3,8 @@
 import '@ciscospark/plugin-avatar';
 import '@ciscospark/plugin-conversation';
 import '@ciscospark/plugin-flag';
+import '@ciscospark/plugin-mercury';
+import '@ciscospark/plugin-wdm';
```

Comment:
We didn't need them, but we are using them. Is there any cost/benefit to not having them? @ianwremmel?